client/packages/common/src/hooks/useUrlQuery/useHandleUrlQueryParams.ts

```diff
@@ -0,0 +1,65 @@
+import { useUrlQuery } from './useUrlQuery';
+import { Column } from '@openmsupply-client/common';
+import { FilterController } from '../useQueryParams';
+
+// This hook uses the state of the url query parameters to provide query parameters and update methods to tables.
+
+export const useHandleUrlQueryParams = (filterIndex: string) => {
```

Comment:
Passing in filterIndex when instantiating hook, as it never changes per table, and it's not super easy to find out what it is when updating the filter query.

Reply:
when I first read through, I thought that this would be numeric, because it's called 'index'. I see now that it is the property name.. perhaps `filterKey` would be clearer?

Reply:
Huh, I could have sworn I got the name `filterIndex` from somewhere else... but apparently not. 🤷‍♂️

Changed to `filterKey` (along with `initialSortByKey` as suggested elsewhere)